test/dotnet-new3.UnitTests/Approvals/DotnetNewList.BasicTest_WhenLegacyCommandIsUsed_common.verified.txt

```diff
@@ -19,6 +19,8 @@ dotnet gitignore file                         gitignore                  Config
 Dotnet local tool manifest file               tool-manifest              Config                
 EditorConfig file                             editorconfig               Config                
 global.json file                              globaljson                 Config                
+MSBuild Directory.Build.props file            buildprops     MSBuild     MSBuild/props         
+MSBuild Directory.Build.targets file          buildtargets   MSBuild     MSBuild/props         
```

Comment:
`MSBuild` language was removed from template, but not from verification (same below).